pydis_site/apps/api/migrations/0051_import_whitelisted_items.py

```diff
@@ -0,0 +1,77 @@
+# Generated by Django 2.2.6 on 2019-11-17 18:21
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    def create_whitelist(app, schema_editor):
+        Whitelist = app.get_model("api", "Whitelist")
+        invites_whitelist = [
```

Comment:
Let's update this migration with the latest configuration from the bot before this PR is merged.